jib-cli/src/integration-test/resources/jarTest/spring-boot/spring-boot/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'org.springframework.boot' version '2.3.7.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'java'
+}
+
+sourceCompatibility = '1.8'
+
+bootJar {
+    layered {
+        enabled = true
+    }
+}
```

Comment:
Shouldn't this be removed?

Reply:
Hrm interesting, yes, but then it doesn't seem to matter if the build is layered or not

Reply:
I wonder why ` assertThat(jarFile.getEntry("BOOT-INF/layers.idx")).isNull();` doesn't fail when we have it.

Reply:
@loosebazooka are you running the `./gradlew integT` task? They don't run in non-integration tests.

Reply:
pretty sure I am, is it failing for you?

Reply:
oh lol that directory isn't used. It was some artifact of the copy. Sorry.